e2e/cypress/support/api/common.js

```diff
@@ -21,6 +21,16 @@ Cypress.Commands.add('apiUploadFile', (name, filePath, options = {}) => {
         });
 });
 
+Cypress.Commands.add('apiDownloadFileAndVerifyContentType', (fileURL, contentType = 'application/zip') => {
+    cy.request(fileURL).then((response) => {
+        // * Verify the download
+        expect(response.status).to.equal(200);
+
+        // * Confirm it's a zip file
```

Comment:
Per this command implementation, `contentType` may be different. Should this be?
```suggestion
        // * Confirm its content type
```